docs/book/quick-start.md

```diff
@@ -84,29 +84,82 @@ If you are at all uncertain about the source of a variable in your view, escape
 
 Routes determine which controller to call based on the URI and other information from the request.
 
-Configure a route and a controller in `module/Application/config/module.config.php`:
+A couple of array values are declared for routes and controllers are added to  `module/Application/config/module.config.php` as shown below:
```

Comment:
I believe that this rephrasing makes things more confusing. What problem are we attempting to solve in the original sentence?

Reply:
It should be pointed out here that the existing module configuration file must be extended.

---

Excerpt from the documentation writer's guide (working draft):

> ### Filenames
> 
> A filename must be placed before a code example because there are always two parts for this information: 
> 
> 1. what kind of file is this
> 2. and what is the name or path.
> 
> Both informations in a code comment would be too long and misplaced.
> 
> #### Examples
> 
> > Create a file for the module configuration called `module.config.php` under `laminas-mvc-tutorial/module/Album/config/`:
> 
> > …in a factory, e.g. `src/Album/Controller/AlbumControllerFactory.php`:
> 
> > …in a module configuration file, e.g. `module/Album/config/module.config.php`: